conf/jasminegraph-server.properties

```diff
@@ -25,20 +25,33 @@ org.jasminegraph.server.nworkers=2
 org.jasminegraph.server.npartitions=2
 #org.jasminegraph.server.runtime.location is the location where temporary files of JasmineGraph master is kept
 org.jasminegraph.server.runtime.location=./tmp
-org.jasminegraph.worker.path=/var/tmp/jasminegraph/
+org.jasminegraph.worker.path=/var/tmp/jasminegraph_fl/
 #Path to keep jasminegraph artifacts in order to copy them to remote locations. If this is not set then the artifacts in the
 #JASMINEGRAPH_HOME location will be copied instead.
 #if it is empty please leave a space, else it will cause an issue
 org.jasminegraph.artifact.path=
 #org.jasminegraph.partitioner.metis.bin is the location where the METIS graph partitioner's gpmetis executable is installed
-org.jasminegraph.partitioner.metis.bin=/home/ubuntu/software/metis-5.1.0/bin
+org.jasminegraph.partitioner.metis.bin=/home/damitha/ubuntu/software/metis/metis-5.1.0/bin
```

Comment:
We do not need to change the existing Metis path

Reply:
Fixed in commit [](https://github.com/miyurud/jasminegraph/pull/109/commits/8c750f47042b2f49312a70fb3289dd9aa9169408))

Reply:
Fixed in commit 8c750f4

Reply:
Done